src/metabase/events/recent_views.clj

```diff
@@ -0,0 +1,49 @@
+(ns metabase.events.recent-views
```

Comment:
This namespace is mainly a few parts copied over from `metabase.events.view-log`, adapted to write to the new `recent_views` table. It's still (also) writing to the view log but I'll remove that in a follow-up.